assets/css/_global.css

```diff
@@ -74,7 +74,7 @@ a {
   text-decoration: none;
 }
 
-a:not(.button) {
+a:not(.button, .resource-block) {
```

Comment:
I noticed there's another rule just for `.resource-block` what do you need to inherit from here?

Reply:
@mordecai174 the resource blocks are all anchor links; the base link styles overwrite them.